src/h5web/providers/utils.ts

```diff
@@ -112,6 +112,10 @@ export function assertSimpleShape<T extends HDF5Type>(
   }
 }
 
+export function getLink(name: string, group: HDF5Group): HDF5Link | undefined {
+  return group.links?.find((l) => l.title === name);
+}
```

Comment:
I need this to get to the ID of the `"errors"` dataset. We don't need to retrieve the whole dataset to get the ID. I may remove this in my next PR, though, since I may end up retrieving the errors dataset to assert on its shape/type.

Reply:
You may need it for `title` as well

Reply:
🎣 I made the same temporary fix.